social-network/src/Article.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import './Article.css';
+import './Author';
+
+
+export default class Article extends React.Component{
+   
+    render(){
+        //console.log(this.props.indArticle);
+        let months = ['Jan','Feb','Mar','Apr', 'May', 'Jun', 'Jul', 'Aug','Sep','Oct','Nov','Dec'];
+        let tmpMonth = this.props.indArticle.postedDate.substring(5,7);
```

Comment:
Clever way of getting the month. Another way is to parse using the Date object and use a `getMonth` function. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth